step-by-step-general/03-get-files.md

```diff
@@ -13,17 +13,29 @@ The latest stable version may differ from the one shown below.
 Do **NOT** try to find the manager version shown in the image, it might be outdated.
 !!!
 ===
+2. In patcher tab, tap Select an Application
+3. Find the app that you want to patch.
+!!! If you cannot find the app...
+If you have not installed the app on your phone, the package name would be shown instead.
+These would be indicated by a missing icon. You can find a lookup table [here](/05-versions.md).
+You can also find the package name for an app by searching "\<app name\> package name" in a browser.
 
-2. Google search the app you wanted to patch and the version required. Example: "youtube music 6.10.52 apk".
-!!!warning
-**Do NOT install the downloaded APK.**
+If you are certain that you have the app installed, using the search bar may help.
 !!!
-!!!warning
-**The app from Google Play may NOT work.** You may or may not be able to patch the one from Google Play. Use the downloaded one.
+4. The suggested version of the app would be shown. 
+!!! If it said "Any version"
+This means that nobody has reported a version that did not work *yet*.
+It means that you can *try* the latest version of apk available.
+In case it doesn't work, go back a few versions and check if it works, and repeat if it doesn't.
+You can also report back the version that works for you at [Discord](https://discord.gg/revanced) or [Github (if nobody else has reported)](https://github.com/ReVanced/revanced-patches/issues)
 !!!
-3. Get the APK, APKMirror usually has them.
-4. Verify that it said "DOWNLOAD APK" not "DOWNLOAD APK BUNDLE". The file should end with `.apk` not `.apks` or `.apkm`.
+5. You can tap the version shown to open an online search result of the suggested apk. You can also just google it.
+6. If there are multiple variants, you most likely need the one marked `arm64-v8a` and `APK` (not bundle).
```

Comment:
Combining the arch information with the file type can cause confusion. Separating them may be simpler:

```suggestion
6. If there are multiple variants, choose the one marked `APK` (not bundle) with the "arch" marked as universal. If no universal arch is available, you most likely need the `arm64-v8a` or `armeabi-v7a` one.
```